http://bazaar.launchpad.net/~squid/squid/3-trunk/revision/13144

    Committer: Amos Jeffries
    Date: 2013-11-23 01:28:52 UTC
    Revision ID: squid3@treenet.co.nz-20131123012852-zpe8hamjrs5vy42w

http://bugs.squid-cache.org/show_bug.cgi?id=3970
Bug 3970: max_filedescriptors disabled due to missing setrlimit

=== modified file 'configure.ac'
--- configure.ac	2013-11-16 13:19:28 +0000
+++ configure.ac	2013-11-23 01:28:52 +0000
@@ -2534,7 +2534,6 @@
     ;;
 esac
 
-
 dnl --with-maxfd present for compatibility with Squid-2.
 dnl undocumented in ./configure --help  to encourage using the Squid-3 directive
 AC_ARG_WITH(maxfd,,
@@ -2542,6 +2541,7 @@
   case ${withval} in
     [[0-9]]*)
       squid_filedescriptors_num=$withval
+      AC_MSG_NOTICE([forcing default of $squid_filedescriptors_num filedescriptors (user-forced)])
       ;;
     *)
       AC_MSG_ERROR(--with-maxfd expects a numeric argument)
@@ -2556,6 +2556,7 @@
   case ${withval} in
     [[0-9]]*)
       squid_filedescriptors_num=$withval
+      AC_MSG_NOTICE([forcing default of $squid_filedescriptors_num filedescriptors (user-forced)])
       ;;
     *)
       AC_MSG_ERROR(--with-filedescriptors expects a numeric argument)
@@ -2564,10 +2565,9 @@
 ])
 
 SQUID_CHECK_DEFAULT_FD_SETSIZE
-if test "x$squid_filedescriptors_num" = "x"; then
-  SQUID_CHECK_MAXFD
-else
-  AC_MSG_NOTICE([forcing use of $squid_filedescriptors_num filedescriptors (user-forced)])
+SQUID_CHECK_MAXFD
+if test "x$squid_filedescriptors_num" != "x"; then
+  AC_MSG_NOTICE([Default number of fieldescriptors: $squid_filedescriptors_num])
 fi
 if test "$squid_filedescriptors_num" -lt 512 ; then
     AC_MSG_WARN([$squid_filedescriptors_num may not be enough filedescriptors if your])

